packages/engine/errors/2571.md

```diff
@@ -0,0 +1,6 @@
+---
+original: "Object is of type 'unknown'."
+excerpt: "This variable is unknown so I don't know what type it or if is in it."
+---
+
+This variable is unknown so I don't know what type it or if is in it.
```

Comment:
```suggestion
excerpt: "This variable is unknown so I don't know what type it or if is in it."
---

This variable is unknown so I don't know what type it or if is in it.
```

I don't think duplicating the error translation is a good thing, the excerpt should be a short description of the error but the content should be a full explanation, not the exact same thing.

Also, the sentence isn't grammarly correct.
```
what type it or if is in it.
```